setup.py

```diff
@@ -61,29 +90,12 @@ def get_version(*file_paths):
         'edx_proctoring',
     ],
     include_package_data=True,
-    install_requires=[
-        "Django>=1.11,<2.0",
-        "django-model-utils>=2.3.1",
-        "edx-drf-extensions",
-        "djangorestframework>=3.1,<3.7",
-        "django-ipware>=1.1.0",
-        "edx-opaque-keys>=0.4",
-        "pytz>=2018",
-        "pycryptodomex>=3.4.7",
-        "python-dateutil>=2.1",
-        "requests",
-        "stevedore",
-        "six",
-    ],
-    dependency_links=[
-        "git+https://github.com/edx/event-tracking.git@0.2.2#egg=event-tracking==0.2.2",
-    ],
+    install_requires=load_requirements("requirements/base.in"),
```

Comment:
For completeness' sake, should there be a ``tests_require``?

Reply:
...And then maybe integrate the [pytest runner](https://docs.pytest.org/en/latest/goodpractices.html#integrating-with-setuptools-python-setup-py-test-pytest-runner) here?